### ISSUE TITLE
Clicking on a boundary should only offer an attachment, not a rewrite

### ISSUE THREAD

**Issue by jamievicary** [homotopy-io/homotopy-rs#684]
In this workspace clicking at the top boundary of the displayed diagram offers both attachments and rewrites. But only attachments should be offered.

[homotopy_io_export.zip](https://github.com/homotopy-io/homotopy-rs/files/9352642/homotopy_io_export.zip)

**Comment by calintat** [comment]
Basically the same issue as #4.

**Comment by calintat** [comment]
@jamievicary I think I may have unintentionally fixed this in 27efe859c27146b2566cbfa4454252f0e2c5c97e but I'm not entirely sure.

I checked what you said in https://github.com/homotopy-io/homotopy-rs/issues/4#issuecomment-828711850 and that is already how we search for attachments. I believe the problem was that when you click near the boundary, that matches a simplex with two points on the boundary and one point in the interior. Previously, we searched for attachments for every point of the simplex, but what my commit does is only search for attachments for the "source" point of the simplex which if you clicked near the boundary will be on the boundary.

I can't really verify this because I can't import any of the workspaces in #4 as they're too old, and I don't really understand what the problem in this workspace is. It already works like you described in your comment.
